### PR TITLE
Don't default classifications if colors have been customized

### PR DESCRIPTION
**Pull request by JoeRobich**
Fix for  https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1128270

### CODE REVIEW

File: src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.ForegroundColorDefaulter.cs [hunk 1]
```diff
@@ -259,6 +259,13 @@ public void DefaultClassifications()
                     return;
                 }
 
+                // If the theme is customized, then do not change the classification colors.
+                if (isThemeCustomized)
+                {
+                    _settings.HasThemeBeenDefaulted[themeId] = true;
```
Comment (ryzngard) [inline src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.ForegroundColorDefaulter.cs:265]: it seems a little odd that `isThemeCustomized` makes `HasThemeBeenDefaulted` as true. Should we remove this if and just add a check in the if above? Does this setting need to be set?

Comment (JoeRobich) [reply src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.ForegroundColorDefaulter.cs:265]: There is no reason to try and default colors in the future either.

Comment (ryzngard) [reply src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.ForegroundColorDefaulter.cs:265]: Yea, but `HasThemeBeenDefaulted` seems misleading in this case. Maybe we should rename that? Idk if it's a breaking change, but it just seems weird when looking and could lead to future bugs if the intent of the setting is misunderstood.

Comment (JoeRobich) [reply src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.ForegroundColorDefaulter.cs:265]: What if I rename it all to "SkipThemeDefaulting"

**Pull request merged**